examples/timestamping/backend/src/api.rs

```diff
@@ -21,28 +21,39 @@ use schema::{Schema, TimestampEntry};
 use transactions::TxTimestamp;
 use TIMESTAMPING_SERVICE;
 
+/// The structure describes the query parameters for `handle_timestamp`
+/// and `handle_timestamp_proof` endpoints.
 #[derive(Debug, Clone, Copy, Serialize, Deserialize)]
 pub struct TimestampQuery {
+    /// The hash of the given timestamp.
     pub hash: Hash,
 }
 
 impl TimestampQuery {
+    /// Creates new `TimestampQuery` with given `hash`.
     pub fn new(hash: Hash) -> Self {
         TimestampQuery { hash }
     }
 }
 
+/// The structure describes the information to prove the correctness of
+/// timestamp entry.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct TimestampProof {
+    /// The `BlockProof` of the appropriate block.
```

Comment:
What block is `appropriate`? Perhaps `last`? I'd also remove every `The` at the beginning here and below. @MartaDubno do we have any policy about `The` in such position?